### PR TITLE
Review section subsection titles

### PR DESCRIPTION
**Pull request by mariaruth**
Reviewed titles of all chapters, sections and subsections and proposed edits.

### CODE REVIEW

File: chapters/2-collaboration.tex [hunk 1]
```diff
@@ -417,7 +417,7 @@ \subsection{Simplifying work across file formats}
 This decision will also affect the version control systems that your team can use.
 
 
-\subsection{Setting up a version control system}
+\subsection{Setting up version control}
```
Comment (kbjarkefur) [inline chapters/2-collaboration.tex:420]:
```suggestion
\subsection{Using version control}
```
Minor, but I think "using" is more appropriate than "setting up" as you use systems that are already set up

**Review by kbjarkefur — APPROVED** [review]

**Comment by kbjarkefur** [comment]
Minor suggestion but otherwise I think all of this is really great!

File: chapters/7-publication.tex [hunk 2]
```diff
@@ -229,7 +229,7 @@ \subsection{Getting started with {\LaTeX} as a team}
 
 %----------------------------------------------------
 
-\section{Preparing and publishing data}
+\section{Preparing research data}
```
Comment (luizaandrade) [inline chapters/7-publication.tex:232]:
```suggestion
\section{Preparing research data for publication}
```
"Preparing research data" could be read as preparing for analysis, as in data processing.

File: chapters/7-publication.tex [hunk 3]
```diff
@@ -338,7 +338,7 @@ \subsection{De-identifying data for publication}
 such as a computational reproducibility check required for publication,
 if done under careful data security protocols and approved by an IRB.
 
-\subsection{Publishing clean and constructed datasets}
+\subsection{Publishing analysis datasets}
```
Comment (luizaandrade) [inline chapters/7-publication.tex:341]:
```suggestion
\subsection{Publishing research datasets}
```
I think we talk both about the final analysis datasets and the clean data when it's self-collected?

File: chapters/6-analysis.tex [hunk 4]
```diff
@@ -691,7 +691,7 @@ \section{Preparing dynamic documents}
 inclusion of inputs in a single integrated process,
 so you can skip the copying and pasting altogether.
 
-\subsection{Dynamic exploratory analysis}
+\subsection{Doing dynamic exploratory analysis}
```
Comment (luizaandrade) [inline chapters/6-analysis.tex:694]:
```suggestion
\subsection{Conducting dynamic exploratory analysis}
```

I seem to have something against the verb to do.

File: chapters/4-acquisition.tex [hunk 5]
```diff
@@ -49,7 +49,7 @@ \section{Acquiring data ethically and reproducibly}
 Writing down and agreeing to specific details is a good way of doing that.
 
 
-\subsection{Data ownership}
+\subsection{Clarifying data ownership}
```
Comment (bbdaniels) [inline chapters/4-acquisition.tex:52]:
```suggestion
\subsection{Determining data ownership}
```

File: chapters/4-acquisition.tex [hunk 6]
```diff
@@ -128,7 +128,7 @@ \subsection{Data ownership}
 used to acquire the data specifically detail those future uses.
 
 
-\subsection{Data licensing}
+\subsection{Establishing a data license}
```
Comment (bbdaniels) [inline chapters/4-acquisition.tex:131]:
```suggestion
\subsection{Obtaining data licenses}
```

File: chapters/4-acquisition.tex [hunk 7]
```diff
@@ -179,7 +179,7 @@ \subsection{Data licensing}
 so that the intended processes for handling the data are clear.
 
 
-\subsection{Receiving data from development partners}
+\subsection{Documenting data received from partners	}
```
Comment (bbdaniels) [inline chapters/4-acquisition.tex:182]:
```suggestion
\subsection{Documenting data received from partners}
```

File: chapters/6-analysis.tex [hunk 8]
```diff
@@ -95,7 +95,7 @@ \section{Creating analysis datasets}
 will result in cleaner, more efficient, and less error-prone analytical code than if
 you started from a single analysis dataset and repeatedly transformed it.
 
-\subsection{Fitting construction into the data workflow}
+\subsection{Organizing data analysis workflow}
```
Comment (bbdaniels) [inline chapters/6-analysis.tex:98]:
```suggestion
\subsection{Organizing data analysis workflows}
```

**Review by bbdaniels — APPROVED** [review]
Added some changes in commits and suggested a final few to accept before merging. This looks great!

**Pull request merged**